### ISSUE TITLE
tidyr field in DESCRIPTION needs updating 

### ISSUE THREAD

**Issue by aeklant** [topepo/tidyposterior#15]
I tried running the example 

#### I.E.

```r
library(tidyposterior)
# See ? precise_example
data(precise_example)

# Get classification accuracy results for analysis

library(dplyr)
accuracy <- precise_example %>%
   select(id, contains("Accuracy")) %>%
   setNames(tolower(gsub("_Accuracy$", "", names(.)))) 
accuracy

# Model the accuracy results
acc_model <- perf_mod(accuracy, seed = 13311, verbose = FALSE 
```

Got the following error: 

```r
Error in mutate_impl(.data, dots) : 
  Evaluation error: object 'statistic' not found.
```

I was able to trace the error to the version of tidyr, so here are my findings:

* tidyr 0.6.2      - Doesn't work
* tidyr >= 0.7.1- Works

Not sure if 0.7.1 is the minimal version (0.7.0 would be my guess), but I think the DESCRIPTION file needs to be updated.

**Comment by aeklant** [comment]
I wouldn't mind contributing a PR for this, I'm just not sure what the minimal version should be.

**Comment by topepo** [comment]
A full `sessionInfo()` would help.

**Comment by aeklant** [comment]
This is the `session_info()` when I get an error:

```r
> sessioninfo::session_info()
─ Session info ────────────────────────────────────────────────────────────────────────────────────
 setting  value                       
 version  R version 3.4.0 (2017-04-21)
 os       Ubuntu 16.04.2 LTS          
 system   x86_64, linux-gnu           
 ui       RStudio                     
 language (EN)                        
 collate  en_US.UTF-8                 
 tz       Zulu                        
 date     2018-03-08                  

─ Packages ────────────────────────────────────────────────────────────────────────────────────────
 package       * version  date       source        
 assertthat      0.2.0    2017-04-11 CRAN (R 3.4.0)
 base64enc       0.1-3    2015-07-28 CRAN (R 3.4.0)
 bayesplot       1.4.0    2017-09-12 CRAN (R 3.4.0)
 bindr           0.1      2016-11-13 CRAN (R 3.4.0)
 bindrcpp      * 0.2      2017-06-17 CRAN (R 3.4.0)
 broom         * 0.4.2    2017-02-13 CRAN (R 3.4.0)
 class           7.3-14   2015-08-30 CRAN (R 3.2.2)
 clisymbols      1.2.0    2017-05-21 CRAN (R 3.4.0)
 codetools       0.2-14   2015-07-15 CRAN (R 3.2.1)
 colorspace      1.3-2    2016-12-14 CRAN (R 3.4.0)
 colourpicker    1.0      2017-09-27 CRAN (R 3.4.0)
 crosstalk       1.0.0    2016-12-21 CRAN (R 3.4.0)
 curl          * 2.8.1    2017-07-21 cran (@2.8.1) 
 CVST            0.2-1    2013-12-10 CRAN (R 3.4.0)
 ddalpha         1.3.1.1  2018-02-02 CRAN (R 3.4.0)
 DEoptimR        1.0-8    2016-11-19 CRAN (R 3.4.0)
 digest          0.6.12   2017-01-27 CRAN (R 3.4.0)
 dimRed          0.1.0    2017-05-04 CRAN (R 3.4.0)
 dplyr         * 0.7.4    2017-09-28 CRAN (R 3.4.0)
 DRR             0.0.3    2018-01-06 CRAN (R 3.4.0)
 DT              0.2      2016-08-09 CRAN (R 3.4.0)
 dygraphs        1.1.1.4  2017-01-04 CRAN (R 3.4.0)
 foreign         0.8-66   2015-08-19 CRAN (R 3.2.2)
 ggplot2       * 2.2.1    2016-12-30 CRAN (R 3.4.0)
 glue            1.2.0    2017-10-29 CRAN (R 3.4.0)
 gower           0.1.2    2017-02-23 CRAN (R 3.4.0)
 gridExtra       2.3      2017-09-09 CRAN (R 3.4.0)
 gtable          0.2.0    2016-02-26 CRAN (R 3.4.0)
 gtools          3.5.0    2015-05-29 CRAN (R 3.4.0)
 htmltools       0.3.6    2017-04-28 CRAN (R 3.4.0)
 htmlwidgets     0.9      2017-07-10 CRAN (R 3.4.0)
 httpuv          1.3.5    2017-07-04 cran (@1.3.5) 
 igraph          1.1.2    2017-07-21 CRAN (R 3.4.0)
 inline          0.3.14   2015-04-13 CRAN (R 3.4.0)
 ipred           0.9-6    2017-03-01 CRAN (R 3.4.0)
 kernlab         0.9-25   2016-10-03 CRAN (R 3.4.0)
 lattice         0.20-33  2015-07-14 CRAN (R 3.2.1)
 lava            1.6      2018-01-13 CRAN (R 3.4.0)
 lazyeval        0.2.0    2016-06-12 CRAN (R 3.4.0)
 lme4            1.1-13   2017-04-19 CRAN (R 3.4.0)
 loo             1.1.0    2017-03-27 CRAN (R 3.4.0)
 lubridate       1.6.0    2016-09-13 CRAN (R 3.4.0)
 magrittr        1.5      2014-11-22 CRAN (R 3.4.0)
 markdown        0.8      2017-04-20 CRAN (R 3.4.0)
 MASS            7.3-45   2015-11-10 CRAN (R 3.2.2)
 Matrix          1.2-3    2015-11-28 CRAN (R 3.2.2)
 matrixStats     0.53.1   2018-02-11 CRAN (R 3.4.0)
 mime            0.5      2016-07-07 CRAN (R 3.4.0)
 miniUI          0.1.1    2016-01-15 CRAN (R 3.4.0)
 minqa           1.2.4    2014-10-09 CRAN (R 3.4.0)
 mnormt          1.5-5    2016-10-15 CRAN (R 3.4.0)
 munsell         0.4.3    2016-02-13 CRAN (R 3.4.0)
 nlme            3.1-124  2016-01-20 CRAN (R 3.2.3)
 nloptr          1.0.4    2014-08-04 CRAN (R 3.4.0)
 nnet            7.3-12   2016-02-02 CRAN (R 3.2.3)
 pkgconfig       2.0.1    2017-03-21 CRAN (R 3.4.0)
 plyr            1.8.4    2016-06-08 CRAN (R 3.4.0)
 prodlim         1.6.1    2017-03-06 CRAN (R 3.4.0)
 psych           1.7.5    2017-05-03 CRAN (R 3.4.0)
 purrr           0.2.4    2017-10-18 CRAN (R 3.4.0)
 R6              2.2.2    2017-06-17 cran (@2.2.2) 
 Rcpp            0.12.13  2017-09-28 CRAN (R 3.4.0)
 RcppRoll        0.2.2    2015-04-05 CRAN (R 3.4.0)
 recipes         0.1.2    2018-01-11 CRAN (R 3.4.0)
 reshape2        1.4.2    2016-10-22 CRAN (R 3.4.0)
 rlang           0.2.0    2018-02-20 CRAN (R 3.4.0)
 robustbase      0.92-8   2017-11-01 CRAN (R 3.4.0)
 rpart           4.1-10   2015-06-29 CRAN (R 3.2.1)
 rsample         0.0.2    2017-11-12 CRAN (R 3.4.0)
 rsconnect       0.8.5    2017-08-23 CRAN (R 3.4.0)
 rstan           2.17.3   2018-01-20 CRAN (R 3.4.0)
 rstanarm        2.17.3   2018-02-18 CRAN (R 3.4.0)
 rstantools      1.4.0    2017-12-21 CRAN (R 3.4.0)
 scales          0.4.1    2016-11-09 CRAN (R 3.4.0)
 sessioninfo     1.0.0    2017-06-21 CRAN (R 3.4.0)
 sfsmisc         1.1-1    2017-06-08 CRAN (R 3.4.0)
 shiny           1.0.5    2017-08-23 cran (@1.0.5) 
 shinyjs         1.0      2018-01-08 CRAN (R 3.4.0)
 shinystan       2.4.0    2017-08-02 CRAN (R 3.4.0)
 shinythemes     1.1.1    2016-10-12 CRAN (R 3.4.0)
 StanHeaders     2.17.2   2018-01-20 CRAN (R 3.4.0)
 stringi         1.1.5    2017-04-07 CRAN (R 3.4.0)
 stringr         1.2.0    2017-02-18 CRAN (R 3.4.0)
 survival        2.41-3   2017-04-04 CRAN (R 3.4.0)
 threejs         0.3.1    2017-08-13 CRAN (R 3.4.0)
 tibble          1.3.4    2017-08-22 cran (@1.3.4) 
 tidyposterior * 0.0.1    2017-11-14 CRAN (R 3.4.0)
 tidyr           0.6.2    2017-05-04 CRAN (R 3.4.0)
 tidyselect      0.2.4    2018-02-26 CRAN (R 3.4.0)
 timeDate        3043.102 2018-02-21 CRAN (R 3.4.0)
 withr           1.0.2    2016-06-20 CRAN (R 3.4.0)
 xtable          1.8-2    2016-02-05 CRAN (R 3.4.0)
 xts             0.10-1   2017-12-20 CRAN (R 3.4.0)
 zoo             1.8-0    2017-04-12 CRAN (R 3.4.0)
```
If I update `tidyr` to the newest version, the error goes away. This is what the `session_info()` looks like in that case: 

```r
> sessioninfo::session_info()
─ Session info ────────────────────────────────────────────────────────────────────────────────────
 setting  value                       
 version  R version 3.4.0 (2017-04-21)
 os       Ubuntu 16.04.2 LTS          
 system   x86_64, linux-gnu           
 ui       RStudio                     
 language (EN)                        
 collate  en_US.UTF-8                 
 tz       Zulu                        
 date     2018-03-08                  

─ Packages ────────────────────────────────────────────────────────────────────────────────────────
 package       * version  date       source        
 assertthat      0.2.0    2017-04-11 CRAN (R 3.4.0)
 base64enc       0.1-3    2015-07-28 CRAN (R 3.4.0)
 bayesplot       1.4.0    2017-09-12 CRAN (R 3.4.0)
 bindr           0.1      2016-11-13 CRAN (R 3.4.0)
 bindrcpp      * 0.2      2017-06-17 CRAN (R 3.4.0)
 broom         * 0.4.2    2017-02-13 CRAN (R 3.4.0)
 class           7.3-14   2015-08-30 CRAN (R 3.2.2)
 clisymbols      1.2.0    2017-05-21 CRAN (R 3.4.0)
 codetools       0.2-14   2015-07-15 CRAN (R 3.2.1)
 colorspace      1.3-2    2016-12-14 CRAN (R 3.4.0)
 colourpicker    1.0      2017-09-27 CRAN (R 3.4.0)
 crosstalk       1.0.0    2016-12-21 CRAN (R 3.4.0)
 curl          * 2.8.1    2017-07-21 cran (@2.8.1) 
 CVST            0.2-1    2013-12-10 CRAN (R 3.4.0)
 ddalpha         1.3.1.1  2018-02-02 CRAN (R 3.4.0)
 DEoptimR        1.0-8    2016-11-19 CRAN (R 3.4.0)
 digest          0.6.12   2017-01-27 CRAN (R 3.4.0)
 dimRed          0.1.0    2017-05-04 CRAN (R 3.4.0)
 dplyr         * 0.7.4    2017-09-28 CRAN (R 3.4.0)
 DRR             0.0.3    2018-01-06 CRAN (R 3.4.0)
 DT              0.2      2016-08-09 CRAN (R 3.4.0)
 dygraphs        1.1.1.4  2017-01-04 CRAN (R 3.4.0)
 foreign         0.8-66   2015-08-19 CRAN (R 3.2.2)
 ggplot2       * 2.2.1    2016-12-30 CRAN (R 3.4.0)
 glue            1.2.0    2017-10-29 CRAN (R 3.4.0)
 gower           0.1.2    2017-02-23 CRAN (R 3.4.0)
 gridExtra       2.3      2017-09-09 CRAN (R 3.4.0)
 gtable          0.2.0    2016-02-26 CRAN (R 3.4.0)
 gtools          3.5.0    2015-05-29 CRAN (R 3.4.0)
 htmltools       0.3.6    2017-04-28 CRAN (R 3.4.0)
 htmlwidgets     0.9      2017-07-10 CRAN (R 3.4.0)
 httpuv          1.3.5    2017-07-04 cran (@1.3.5) 
 igraph          1.1.2    2017-07-21 CRAN (R 3.4.0)
 inline          0.3.14   2015-04-13 CRAN (R 3.4.0)
 ipred           0.9-6    2017-03-01 CRAN (R 3.4.0)
 kernlab         0.9-25   2016-10-03 CRAN (R 3.4.0)
 lattice         0.20-33  2015-07-14 CRAN (R 3.2.1)
 lava            1.6      2018-01-13 CRAN (R 3.4.0)
 lazyeval        0.2.0    2016-06-12 CRAN (R 3.4.0)
 lme4            1.1-13   2017-04-19 CRAN (R 3.4.0)
 loo             1.1.0    2017-03-27 CRAN (R 3.4.0)
 lubridate       1.6.0    2016-09-13 CRAN (R 3.4.0)
 magrittr        1.5      2014-11-22 CRAN (R 3.4.0)
 markdown        0.8      2017-04-20 CRAN (R 3.4.0)
 MASS            7.3-45   2015-11-10 CRAN (R 3.2.2)
 Matrix          1.2-3    2015-11-28 CRAN (R 3.2.2)
 matrixStats     0.53.1   2018-02-11 CRAN (R 3.4.0)
 mime            0.5      2016-07-07 CRAN (R 3.4.0)
 miniUI          0.1.1    2016-01-15 CRAN (R 3.4.0)
 minqa           1.2.4    2014-10-09 CRAN (R 3.4.0)
 mnormt          1.5-5    2016-10-15 CRAN (R 3.4.0)
 munsell         0.4.3    2016-02-13 CRAN (R 3.4.0)
 nlme            3.1-124  2016-01-20 CRAN (R 3.2.3)
 nloptr          1.0.4    2014-08-04 CRAN (R 3.4.0)
 nnet            7.3-12   2016-02-02 CRAN (R 3.2.3)
 pkgconfig       2.0.1    2017-03-21 CRAN (R 3.4.0)
 plyr            1.8.4    2016-06-08 CRAN (R 3.4.0)
 prodlim         1.6.1    2017-03-06 CRAN (R 3.4.0)
 psych           1.7.5    2017-05-03 CRAN (R 3.4.0)
 purrr           0.2.4    2017-10-18 CRAN (R 3.4.0)
 R6              2.2.2    2017-06-17 cran (@2.2.2) 
 Rcpp            0.12.13  2017-09-28 CRAN (R 3.4.0)
 RcppRoll        0.2.2    2015-04-05 CRAN (R 3.4.0)
 recipes         0.1.2    2018-01-11 CRAN (R 3.4.0)
 reshape2        1.4.2    2016-10-22 CRAN (R 3.4.0)
 rlang           0.2.0    2018-02-20 CRAN (R 3.4.0)
 robustbase      0.92-8   2017-11-01 CRAN (R 3.4.0)
 rpart           4.1-10   2015-06-29 CRAN (R 3.2.1)
 rsample         0.0.2    2017-11-12 CRAN (R 3.4.0)
 rsconnect       0.8.5    2017-08-23 CRAN (R 3.4.0)
 rstan           2.17.3   2018-01-20 CRAN (R 3.4.0)
 rstanarm        2.17.3   2018-02-18 CRAN (R 3.4.0)
 rstantools      1.4.0    2017-12-21 CRAN (R 3.4.0)
 scales          0.4.1    2016-11-09 CRAN (R 3.4.0)
 sessioninfo     1.0.0    2017-06-21 CRAN (R 3.4.0)
 sfsmisc         1.1-1    2017-06-08 CRAN (R 3.4.0)
 shiny           1.0.5    2017-08-23 cran (@1.0.5) 
 shinyjs         1.0      2018-01-08 CRAN (R 3.4.0)
 shinystan       2.4.0    2017-08-02 CRAN (R 3.4.0)
 shinythemes     1.1.1    2016-10-12 CRAN (R 3.4.0)
 StanHeaders     2.17.2   2018-01-20 CRAN (R 3.4.0)
 stringi         1.1.5    2017-04-07 CRAN (R 3.4.0)
 stringr         1.2.0    2017-02-18 CRAN (R 3.4.0)
 survival        2.41-3   2017-04-04 CRAN (R 3.4.0)
 threejs         0.3.1    2017-08-13 CRAN (R 3.4.0)
 tibble          1.3.4    2017-08-22 cran (@1.3.4) 
 tidyposterior * 0.0.1    2017-11-14 CRAN (R 3.4.0)
 tidyr           0.8.0    2018-01-29 CRAN (R 3.4.0)
 tidyselect      0.2.4    2018-02-26 CRAN (R 3.4.0)
 timeDate        3043.102 2018-02-21 CRAN (R 3.4.0)
 withr           1.0.2    2016-06-20 CRAN (R 3.4.0)
 xtable          1.8-2    2016-02-05 CRAN (R 3.4.0)
 xts             0.10-1   2017-12-20 CRAN (R 3.4.0)
 zoo             1.8-0    2017-04-12 CRAN (R 3.4.0)
```

And just for completeness. I ran it on a different machine where it also works fine (this is how I figured I needed to update `tidyr`). This is the `session_info` for that:

```r
> sessioninfo::session_info()
─ Session info ──────────────────────────────────────────────────────────────────────────────
 setting  value                       
 version  R version 3.4.1 (2017-06-30)
 os       Windows >= 8 x64            
 system   x86_64, mingw32             
 ui       RStudio                     
 language (EN)                        
 collate  English_Canada.1252         
 tz       America/New_York            
 date     2018-03-08                  

─ Packages ──────────────────────────────────────────────────────────────────────────────────
 package       * version  date       source        
 assertthat      0.2.0    2017-04-11 CRAN (R 3.4.1)
 base64enc       0.1-3    2015-07-28 CRAN (R 3.4.1)
 bayesplot       1.4.0    2017-09-12 CRAN (R 3.4.3)
 bindr           0.1      2016-11-13 CRAN (R 3.4.1)
 bindrcpp      * 0.2      2017-06-17 CRAN (R 3.4.1)
 broom         * 0.4.2    2017-02-13 CRAN (R 3.4.0)
 cellranger      1.1.0    2016-07-27 CRAN (R 3.4.1)
 class           7.3-14   2015-08-30 CRAN (R 3.4.1)
 clisymbols      1.2.0    2017-05-21 CRAN (R 3.4.3)
 codetools       0.2-15   2016-10-05 CRAN (R 3.4.1)
 colorspace      1.3-2    2016-12-14 CRAN (R 3.4.1)
 colourpicker    1.0      2017-09-27 CRAN (R 3.4.3)
 crosstalk       1.0.0    2016-12-21 CRAN (R 3.4.3)
 CVST            0.2-1    2013-12-10 CRAN (R 3.4.3)
 ddalpha         1.3.1.1  2018-02-02 CRAN (R 3.4.3)
 DEoptimR        1.0-8    2016-11-19 CRAN (R 3.4.1)
 digest          0.6.12   2017-01-27 CRAN (R 3.4.1)
 dimRed          0.1.0    2017-05-04 CRAN (R 3.4.3)
 dplyr         * 0.7.2    2017-07-20 CRAN (R 3.4.1)
 DRR             0.0.3    2018-01-06 CRAN (R 3.4.3)
 DT              0.4      2018-01-30 CRAN (R 3.4.3)
 dygraphs        1.1.1.4  2017-01-04 CRAN (R 3.4.3)
 forcats         0.2.0    2017-01-23 CRAN (R 3.4.1)
 foreign         0.8-69   2017-06-22 CRAN (R 3.4.1)
 ggplot2       * 2.2.1    2016-12-30 CRAN (R 3.4.1)
 glue            1.1.1    2017-06-21 CRAN (R 3.4.1)
 gower           0.1.2    2017-02-23 CRAN (R 3.4.3)
 gridExtra       2.3      2017-09-09 CRAN (R 3.4.3)
 gtable          0.2.0    2016-02-26 CRAN (R 3.4.1)
 gtools          3.5.0    2015-05-29 CRAN (R 3.4.1)
 haven           1.1.0    2017-07-09 CRAN (R 3.4.1)
 hms             0.3      2016-11-22 CRAN (R 3.4.1)
 htmltools       0.3.6    2017-04-28 CRAN (R 3.4.1)
 htmlwidgets     1.0      2018-01-20 CRAN (R 3.4.3)
 httpuv          1.3.6.2  2018-03-02 CRAN (R 3.4.3)
 httr            1.3.1    2017-08-20 CRAN (R 3.4.1)
 igraph          1.1.2    2017-07-21 CRAN (R 3.4.3)
 inline          0.3.14   2015-04-13 CRAN (R 3.4.3)
 ipred           0.9-6    2017-03-01 CRAN (R 3.4.3)
 jsonlite        1.5      2017-06-01 CRAN (R 3.4.1)
 kernlab         0.9-25   2016-10-03 CRAN (R 3.4.1)
 lattice         0.20-35  2017-03-25 CRAN (R 3.4.1)
 lava            1.6      2018-01-13 CRAN (R 3.4.3)
 lazyeval        0.2.0    2016-06-12 CRAN (R 3.4.1)
 lme4            1.1-15   2017-12-21 CRAN (R 3.4.3)
 loo             1.1.0    2017-03-27 CRAN (R 3.4.3)
 lubridate       1.6.0    2016-09-13 CRAN (R 3.4.1)
 magrittr        1.5      2014-11-22 CRAN (R 3.4.1)
 markdown        0.8      2017-04-20 CRAN (R 3.4.1)
 MASS            7.3-47   2017-02-26 CRAN (R 3.4.1)
 Matrix          1.2-10   2017-05-03 CRAN (R 3.4.1)
 matrixStats     0.53.1   2018-02-11 CRAN (R 3.4.3)
 mime            0.5      2016-07-07 CRAN (R 3.4.1)
 miniUI          0.1.1    2016-01-15 CRAN (R 3.4.3)
 minqa           1.2.4    2014-10-09 CRAN (R 3.4.3)
 mnormt          1.5-5    2016-10-15 CRAN (R 3.4.1)
 modelr          0.1.1    2017-07-24 CRAN (R 3.4.1)
 munsell         0.4.3    2016-02-13 CRAN (R 3.4.1)
 nlme            3.1-131  2017-02-06 CRAN (R 3.4.1)
 nloptr          1.0.4    2017-08-22 CRAN (R 3.4.3)
 nnet            7.3-12   2016-02-02 CRAN (R 3.4.1)
 pkgconfig       2.0.1    2017-03-21 CRAN (R 3.4.1)
 plyr            1.8.4    2016-06-08 CRAN (R 3.4.1)
 prodlim         1.6.1    2017-03-06 CRAN (R 3.4.3)
 psych           1.7.8    2017-09-09 CRAN (R 3.4.1)
 purrr         * 0.2.3    2017-08-02 CRAN (R 3.4.1)
 R6              2.2.2    2017-06-17 CRAN (R 3.4.1)
 Rcpp            0.12.12  2017-07-15 CRAN (R 3.4.1)
 RcppRoll        0.2.2    2015-04-05 CRAN (R 3.4.3)
 readr         * 1.1.1    2017-05-16 CRAN (R 3.4.1)
 readxl          1.0.0    2017-04-18 CRAN (R 3.4.1)
 recipes         0.1.2    2018-01-11 CRAN (R 3.4.3)
 reshape2        1.4.2    2016-10-22 CRAN (R 3.4.1)
 rlang           0.1.2    2017-08-09 CRAN (R 3.4.1)
 robustbase      0.92-8   2017-11-01 CRAN (R 3.4.3)
 rpart           4.1-11   2017-03-13 CRAN (R 3.4.1)
 rsample         0.0.2    2017-11-12 CRAN (R 3.4.3)
 rsconnect       0.8.5    2017-08-23 CRAN (R 3.4.3)
 rstan           2.17.3   2018-01-20 CRAN (R 3.4.3)
 rstanarm        2.17.3   2018-02-18 CRAN (R 3.4.3)
 rstantools      1.4.0    2017-12-21 CRAN (R 3.4.3)
 rvest           0.3.2    2016-06-17 CRAN (R 3.4.1)
 scales          0.5.0    2017-08-24 CRAN (R 3.4.1)
 sessioninfo     1.0.0    2017-06-21 CRAN (R 3.4.3)
 sfsmisc         1.1-2    2018-03-05 CRAN (R 3.4.3)
 shiny           1.0.5    2017-08-23 CRAN (R 3.4.3)
 shinyjs         1.0      2018-01-08 CRAN (R 3.4.3)
 shinystan       2.4.0    2017-08-02 CRAN (R 3.4.3)
 shinythemes     1.1.1    2016-10-12 CRAN (R 3.4.3)
 StanHeaders     2.17.2   2018-01-20 CRAN (R 3.4.3)
 stringi         1.1.5    2017-04-07 CRAN (R 3.4.1)
 stringr         1.2.0    2017-02-18 CRAN (R 3.4.1)
 survival        2.41-3   2017-04-04 CRAN (R 3.4.1)
 threejs         0.3.1    2017-08-13 CRAN (R 3.4.3)
 tibble        * 1.3.4    2017-08-22 CRAN (R 3.4.1)
 tidyposterior * 0.0.1    2017-11-14 CRAN (R 3.4.3)
 tidyr         * 0.7.1    2017-09-01 CRAN (R 3.4.1)
 tidyselect      0.2.0    2017-08-30 CRAN (R 3.4.1)
 tidyverse     * 1.1.1    2017-01-27 CRAN (R 3.4.1)
 timeDate        3043.102 2018-02-21 CRAN (R 3.4.3)
 withr           2.1.1    2017-12-19 CRAN (R 3.4.3)
 xml2            1.1.1    2017-01-24 CRAN (R 3.4.1)
 xtable          1.8-2    2016-02-05 CRAN (R 3.4.3)
 xts             0.10-1   2017-12-20 CRAN (R 3.4.3)
 zoo             1.8-1    2018-01-08 CRAN (R 3.4.3)
```

TL;DR: It works with `tidyr` version `0.8.0` and `0.7.1`. Doesn't work with version `0.6.2`.

**Comment by topepo** [comment]
I can reproduce this. 

@aeklant I think that I would just change the description file to have `tidyr >= 0.7.1`.
